test/e2e-multicluster-test/multicluster_standalone_test.go

```diff
@@ -184,7 +183,7 @@ var _ = Describe("Test multicluster standalone scenario", func() {
 			g.Expect(k8sClient.List(hubCtx, deploys, client.InNamespace(namespace))).Should(Succeed())
 			g.Expect(len(deploys.Items)).Should(Equal(1))
 			g.Expect(deploys.Items[0].Spec.Replicas).Should(Equal(pointer.Int32(3)))
-		}, 30*time.Second).Should(Succeed())
+		}, 2*time.Minute, time.Second).Should(Succeed())
```

Comment:
do we really need so long time?

Reply:
See reasons above. Yes until we fix them all.